worldeditadditions_commands/commands/measure/mtrig.lua

```diff
@@ -0,0 +1,25 @@
+-- ███    ███ ████████ ██████  ██  ██████
+-- ████  ████    ██    ██   ██ ██ ██
+-- ██ ████ ██    ██    ██████  ██ ██   ███
+-- ██  ██  ██    ██    ██   ██ ██ ██    ██
+-- ██      ██    ██    ██   ██ ██  ██████
+local wea = worldeditadditions
+local v3 = worldeditadditions.Vector3
+worldedit.register_command("mtrig", {
+	params = "",
+	description = "Return the length of and angles of an imginary line between pos1 and pos2 in the selection.",
+	privs = { worldedit = true },
+	require_pos = 2,
+	parse = function(params_text)
+		return true
+	end,
+	func = function(name, params_text)
+		local str = "The measurements of the line pos1,pos2 are Length (D): "
```

Comment:
Perhaps this will read better?
```suggestion
		local str = "The measurements of the line pos1 to pos2 are Length (D): "
```